### DISCUSSION TITLE
ETA for broader Nova language support

### DISCUSSION THREAD

**Discussion by bmaluijb** [deepgram/community#230] [General help] [answered]
Hi,

I love your Nova model as it is fast and cheap. For my use case it will be usable when it supports more languages, like the languages that are currently available or in beta for other models. 

Can you please provide a rough timeline as to when we can expect more language support for Nova?

Thanks!

**Comment by jkroll-deepgram** [comment]
Hi @bmaluijb, glad to hear that Nova is working well for you! Which languages are highest-priority for you? We'd value your feedback to help inform future language expansions.

**Comment by bmaluijb** [comment]
Spanish, Dutch and German are priority. Do you have a rough ETA?

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jkroll-deepgram** [comment, marked as answer]
@bmaluijb We're expecting to launch Spanish Nova later this summer. Dutch and German Nova have a longer horizon, so we still recommend the `general` model with either the `base` (most cost-effective) or `enhanced` (highest accuracy) tier. Some of these model/tier/language combos are in beta as you mentioned; refer to our full [Language docs](https://developers.deepgram.com/docs/language).

We'd be happy to hear any further feedback on your expected usage volume with Nova in additional languages, and what you're looking to optimize for that is not being met by our existing general models - such as accuracy, speed, cost, etc?

**Comment by bmaluijb** [comment]
My top reason for using Nova is speed and low cost. It’s unrivaled at that when compared to other models and other companies.

So I want that, for all languages in my product. I don’t know volumes yet, as I’m building on a startup.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jkroll-deepgram** [comment]
I'm happy to share that we launched Spanish Nova today. You can use it with the params `language=es&model=general&tier=nova`. Let us know if you have any feedback when trying it out!

**Comment by bmaluijb** [comment]
That’s quick, awesome!

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by cwazuidema** [comment]
I'm also waiting for Nova support for Dutch. Is there any news about that?

**Reply by jkroll-deepgram** [reply]
Hi @cwazuidema, we do not have a planned future release for Nova in Dutch. We currently have a beta release (language code `nl`) for the `base` and `enhanced` model tiers. (See [Languages Overview](https://developers.deepgram.com/docs/languages-overview) docs)

**Comment by duanshiqiang** [comment]
I'm also waiting for Chinese support for Nova...

**Comment by jkroll-deepgram** [comment]
@bmaluijb @cwazuidema I'm happy to share that we've recently released another wave of Nova-2 languages. Beyond English dialects, we now support Spanish, Portuguese, Hindi, German, French, Dutch, and Flemish! See our [Models & Languages Overview](https://developers.deepgram.com/docs/models-languages-overview) for the current list along with language codes and model names.
